assets/scripts/game.js

```diff
@@ -0,0 +1,131 @@
+const Player = () => {
+    let marker = "X"
+    const getMarker = () => marker
+    const switchPlayer = () => {
+        if(marker === "X") {
+            marker = "O"
+        } else {
+            marker = "X"
+        }
+    }
+    return { marker, switchPlayer, getMarker }
+}
+
+const GameBoard = (() => {
```

Comment:
For our `GameBoard` let us separate DOM related logic. `GameBoard` is one of the main logic objects of the game and shouldn't be mixed with user interface related stuff.

We should have a separate function/module related to board display which just uses logic coming from `GameBoard`.